### PR TITLE
FT: ZNC-26: add issue template

### PR DESCRIPTION
**Pull request by LaureVergeron**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
+# General support information
+
+GitHub Issues are **reserved** for actionable bug reports (including
+documentation inaccuracies), and feature requests.
+**All questions** (regarding configuration, usecases, performance, community,
```
Comment (bennettbuchanan) [inline .github/ISSUE_TEMPLATE.md]: ~usecases~ use cases

**Review by bennettbuchanan — DISMISSED** [review]
LGTM! :shipit:

**Review by smaffulli — COMMENTED** [review]
LGTM, fixed the typo.

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+- search the issue list for this repository (use the search bar, select
+  "Issues" on the left pane after searching);
+- if there is a duplicate, please do not open your issue, and add a comment
+  to the existing issue instead.
+
+--------------------------------------------------------------------------------
+
+## Bug report information
+
+(delete this section (everything between the lines) if you're not repoting a bug
```
Comment (jonathan-gramain) [inline .github/ISSUE_TEMPLATE.md]: typo: `reporting a bug`

**Review by jonathan-gramain — DISMISSED** [review]

**Review by philipyoo — DISMISSED** [review]

**Review by rahulreddy — APPROVED** [review]

**Review by smaffulli — APPROVED** [review]
Looks good

**Pull request merged**